storagemarket/impl/clientstates/client_fsm.go

```diff
@@ -0,0 +1,105 @@
+package clientstates
+
+import (
+	"github.com/filecoin-project/go-fil-markets/storagemarket"
+	"github.com/filecoin-project/go-statemachine/fsm"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/ipfs/go-cid"
+	"golang.org/x/xerrors"
+)
+
+// ClientEvents are the events that can happen in a storage client
+var ClientEvents = fsm.Events{
+	fsm.Event(storagemarket.ClientEventOpen).
```

Comment:
I don't quite remember why we send an event like this.  Is it because we need an arbitrary event for the FSM to start running?  Doesn't `Begin()` do that?

Reply:
no Begin() does not do that -- it sets an initial state, but there is no implied initial event